### ISSUE TITLE
[FEATURE] Create a template for each movie model

### ISSUE THREAD

**Issue by luismoa** [hikager/yummy_metflix_remake#34]
# Description
Add each page for each movie model; Movie (already done) , Directors and Actors
## Expected result
Add a template view (page) for 
- [ ] Actors
- [ ] Directors
- Some additional information
  - This template should have the same simplicity as in movies template

**For this issue it must be finished before:**

#31 
#32 
#33
